src/Repository/ReferentSpaceAccessInformationRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\ReferentSpaceAccessInformation;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class ReferentSpaceAccessInformationRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, ReferentSpaceAccessInformation::class);
+    }
+
+    public function findByAdherent(Adherent $adherent)
+    {
+        return $this->findOneBy(['adherent' => $adherent]);
```

Comment:
just one idea, what about:
```php
function findByAdherent(Adherent $adherent): ReferentSpaceAccessInformation {
  $obj = $this->findOneBy(['adherent' => $adherent]);
  if (!$obj) {
    $obj = new ReferentSpaceAccessInformation($adherent, new \Date, new \Date);
  }

  return $obj;
}
```

Reply:
Seems strange doing this in a `find` method.
That is cheating😄